### DISCUSSION TITLE
Working with APIs that respond with 202 and a retry time

### DISCUSSION THREAD

**Discussion by nicholasgriffintn** [TanStack/query#7997] [Q&A] [answered]
Hey, I have this API that responds with a 202 accepted if it hasn't processed the data yet, along with a retry time for when it should be available.

What's the best way to get react query to retry at that time if the response is 202?

**Comment by TkDodo** [comment, marked as answer]
implement `refetchInterval` as a function:

```
refetchInterval: (query) => ....
```

**Reply by nicholasgriffintn** [reply]
Ahh yes, I see that now after searching for that name! https://tanstack.com/query/latest/docs/framework/react/examples/auto-refetching

I'll give that a go.
